tests/graph_rust.rs

```diff
@@ -0,0 +1,53 @@
+use graph_rust::{Bfs, Dfs, Graph, GraphType};
+
+#[test]
+fn bfs_non_rec_test() {
+    let g = Graph::default()
+        .with_dir(GraphType::Directed)
+        .with_schema(String::from("..."))
+        .from_file(String::from("data_edges.csv"), None, ",")
+        .build();
+
+    g.bfs(2);
+    println!()
+}
+
+#[test]
+fn dfs_non_rec_test() {
+    let g = Graph::default()
+        .with_dir(GraphType::Directed)
+        .with_schema(String::from("..."))
+        .from_file(String::from("data_edges.csv"), None, ",")
+        .build();
+
+    g.dfs(1);
+    println!()
+}
+
+#[test]
+fn dfs_iter_test() {
+    let g = Graph::default()
+        .with_dir(GraphType::Directed)
+        .with_schema(String::from("..."))
+        .from_file(String::from("data_edges.csv"), None, ",")
+        .build();
+    let mut dfs = Dfs::new(&g, 1);
+    while let Some(node) = dfs.next() {
```

Comment:
Collect the iterator as a vector and compare with expected values. I think it can be random of the bfs/dfs outputs, come up with a solution to test it.